docs/cli/cauldron/add/nativeapp.md

```diff
@@ -14,6 +14,10 @@
 
 * A complete native application descriptor representing the native application version to be added to the Cauldron.
 
+`platformVersion`
```

Comment:
Move to options section instead (`--platformVersion/-p`) per my other comment on the command source.